core/src/env/engine/off_chain/impls.rs

```diff
@@ -153,7 +153,7 @@ impl TypedEnv for EnvInstance {
             .map_err(Into::into)
     }
 
-    fn gas_price<T: EnvTypes>(&mut self) -> Result<T::Balance> {
+    fn gas_price<T: EnvTypes>(&mut self, _gas: u64) -> Result<T::Balance> {
         self.chain_spec
             .gas_price::<T>()
```

Comment:
It would be cool if there was an actual conversion config for the off-chain environment so that the returned amount at least scales linearly to the input.